### DISCUSSION TITLE
most actions on nodes disabled, write protection? How to see, what is protected and what isn't?

### DISCUSSION THREAD

**Discussion by aisbergde** [freeplane/freeplane#1719] [Help]
I try to evaluate freeplane and follow the freeplaneUserGuide
I can't copy the node id, node URI, the node itself or attributes. This happens on all (demo and help) mindmaps:
![image](https://github.com/freeplane/freeplane/assets/9020187/34f7d825-96fb-43d3-bf91-d04cdab8d63a)

Also, a lot of other menu items are disabled:
remove, edit, insert

Is there a write protection?
If so, where to find an indicator about write protection of maps or nodes?
if there is no indicator - could this be a feature request?

**Comment by abc16361** [comment]
Most likely the map is on Viewing mode, so go to menu Edit to check what it is. If it is so, select Editing.

**Comment by euu2021** [comment]
For some reason, the freeplaneUserGuide is created as Read Only:
![image](https://github.com/freeplane/freeplane/assets/77707706/4391fb03-9b0d-4dba-86f7-c90bec3bcf29)

You can use Save As to create a copy and edit it normally.

**Reply by dpolivaev** [reply]
Yes, maps opened from the help menu are opened read-only so that you can not occasionally modify them.
Also, maps opened from read-only files are opened read-only.
